docs/Training-ML-Agents.md

```diff
@@ -435,97 +435,57 @@ behaviors:
   # < Same as above>
 
 parameter_randomization:
-  resampling-interval: 5000
 
   mass:
-    sampler-type: "uniform"
-    min_value: 0.5
-    max_value: 10
+    sampler_type: uniform
+    sampler_parameters:
+        min_value: 0.5
+        max_value: 10
 
-  gravity:
-    sampler-type: "multirange_uniform"
-    intervals: [[7, 10], [15, 20]]
+  length:
+    sampler_type: multirangeuniform
+    sampler_parameters:
+        intervals: [[7, 10], [15, 20]]
 
   scale:
-    sampler-type: "uniform"
-    min_value: 0.75
-    max_value: 3
+    sampler_type: gaussian
+    sampler_parameters:
+        mean: 2
+        st_dev: .3
 ```
 
-Note that `mass`, `gravity` and `scale` are the names of the environment
+Note that `mass`, `length` and `scale` are the names of the environment
```

Comment:
nit: I'm not sure the second half of this comment ("If a parameter specified in the file doesn't...") applies anymore. Maybe rewrite to refer to the SideChannel and/or EnvironmentParameters?